docs/i18n/tr.md

```diff
@@ -71,31 +71,31 @@
 
 <div align="center">
   <sub>Çeviriler:</sub>
-  <a href="docs/i18n/zh_cn.md#readme">
+  <a href="../../docs/i18n/zh_cn.md#readme">
```

Comment:
I would prefer to remove the prefix and just keep `zh_cn.md[...]`, which is equivalent to your solution.

Reply:
Fixed.